camelot/view/action_steps/item_view.py

```diff
@@ -175,34 +162,41 @@ class OpenTableView( UpdateTableView ):
         open the view in a new tab instead of the current tab
         
     """
+
+    new_tab: bool
+    admin_route: Route
```

Comment:
not sure, maar ik denk dat die admin eigenlijk nergens gebruikt w, en dit dus eigenlijk weg mag ?

Reply:
Die admin_route is o.a. nodig voor het aanmaken van de TableView die deze gebruikt om een CollectionProxy aan te maken.